Issue in v2.5.0

![error](https://user-images.githubusercontent.com/11033446/39141467-23c8568e-46e5-11e8-949c-ee854e275c67.png)

I got the error after upgraded to 2.5.0. 

I imported `import * as ReactGA from "react-ga";`. It worked in `2.4.1`, but not in `2.5.0`. Any idea?

The error looks to be caused by the same thing I'm seeing.
If so, I think we can change the issue title to be a bit more specific.

``
node_modules/react-ga/types/index.d.ts(8,8): error TS1192: Module '"/node_modules/@types/react/index"' has no default export.
``

In the recent release, `import React from 'react';` should be `import * as React from 'react';`

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -5,7 +5,7 @@
 //                 Jerry Reptak <https://github.com/jetfault>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import React from 'react';
+import * as React from 'react';
 
 /* tslint:disable no-any */
 

